python/rmm/rmm.py

```diff
@@ -237,6 +237,21 @@ def rmm_cupy_allocator(nbytes):
     return ptr
 
 
+try:
+    from torch.cuda.memory import CUDAPluggableAllocator
+except ImportError:
+    rmm_torch_allocator = None
```

Comment:
Could this bite a user that accesses this attribute and passes it around thinking that it's fine when it's really just None? It might be safer to override `__getattr__` for the module and have it raise an error to prevent the user from accessing this attribute when `CUDAPluggableAllocator` failed to import.

Reply:
Could we alternatively `pass` on `ImportError` to achieve the same effect as defining that module `__getattr__`?

Reply:
I think you'd get close to the same effect, just a slightly less user-friendly version. With a `__getattr__` override you could provide a more friendly error message indicating that this happened because the torch allocator failed to import, whereas if you just avoid defining it the user will see an `AttributeError` without any additional diagnostics and may think it's a bug in rmm.

It's a very minor point though, I'm fine leaving this as is for now and only revisiting in the future if we get a lot of user questions about why the allocator is None.